collectors/plugins.d/plugins_d.c

```diff
@@ -544,53 +545,77 @@ void *pluginsd_worker_thread(void *arg) {
         // get the return code
         int code = mypclose(fp, cd->pid);
 
+        if (likely(code == 0)) {
+            // the plugin reports success
+
+            if (likely(cd->successful_collections)) {
+                sleep((unsigned int) cd->update_every);
+                goto exit_code_analyzed;
+            }
+
+            if(likely(cd->serial_failures <= SERIAL_FAILURES_THRESHOLD)) {
+                infoerr("'%s' (pid %d) does not generate useful output but it reports success (exits with 0). %s.",
```

Comment:
I haven't seen infoerr used anywhere in the code, didn't even know it was defined. Why did you choose to use that one instead of info?

Reply:
Well, I've seen it with the other logging-related macros and either got distracted by its "INFO" label, or that was just a runaway autocomplete. Will change to  `info()`? I don't think this particular message should be an `error()` - after all, it's just a retry.

```c
#define info(args...)    info_int(__FILE__, __FUNCTION__, __LINE__, ##args)
#define infoerr(args...) error_int("INFO", __FILE__, __FUNCTION__, __LINE__, ##args)
#define error(args...)   error_int("ERROR", __FILE__, __FUNCTION__, __LINE__, ##args)
#define fatal(args...)   fatal_int(__FILE__, __FUNCTION__, __LINE__, ##args)
```

Reply:
Changed in to `info()`